test/intl402/Temporal/TimeZone/from/timezone-case-insensitive.js

```diff
@@ -3,10 +3,615 @@
 
 /*---
 esid: sec-temporal.timezone.from
-description: Time zone names are case insensitive
-features: [Temporal]
+description: Time zone identifiers are case-normalized
+features: [Temporal, time-zone-canonicalization]
 ---*/
 
-const timeZone = 'eTc/gMt+1';
-const result = Temporal.TimeZone.from(timeZone);
-assert.sameValue(result.id, 'Etc/GMT+1', `Time zone created from string "${timeZone}"`);
+const timeZoneIdentifiers = [
+  'Africa/Abidjan',
+  'Africa/Algiers',
+  'Africa/Bissau',
+  'Africa/Cairo',
+  'Africa/Casablanca',
+  'Africa/Ceuta',
+  'Africa/El_Aaiun',
+  'Africa/Johannesburg',
+  'Africa/Juba',
+  'Africa/Khartoum',
+  'Africa/Lagos',
+  'Africa/Maputo',
+  'Africa/Monrovia',
+  'Africa/Nairobi',
+  'Africa/Ndjamena',
+  'Africa/Sao_Tome',
+  'Africa/Tripoli',
+  'Africa/Tunis',
+  'Africa/Windhoek',
+  'America/Adak',
+  'America/Anchorage',
+  'America/Araguaina',
+  'America/Argentina/Buenos_Aires',
+  'America/Argentina/Catamarca',
+  'America/Argentina/Cordoba',
+  'America/Argentina/Jujuy',
+  'America/Argentina/La_Rioja',
+  'America/Argentina/Mendoza',
+  'America/Argentina/Rio_Gallegos',
+  'America/Argentina/Salta',
+  'America/Argentina/San_Juan',
+  'America/Argentina/San_Luis',
+  'America/Argentina/Tucuman',
+  'America/Argentina/Ushuaia',
+  'America/Asuncion',
+  'America/Bahia',
+  'America/Bahia_Banderas',
+  'America/Barbados',
+  'America/Belem',
+  'America/Belize',
+  'America/Boa_Vista',
+  'America/Bogota',
+  'America/Boise',
+  'America/Cambridge_Bay',
+  'America/Campo_Grande',
+  'America/Cancun',
+  'America/Caracas',
+  'America/Cayenne',
+  'America/Chicago',
+  'America/Chihuahua',
+  // 'America/Ciudad_Juarez',
```

Comment:
What is special about this one?

Reply:
`America/Ciudad_Juarez` was just added to TZDB this year, and CLDR hasn't caught up yet. So this test would never pass. We can uncomment it if a later Node version includes this ID.  I'll add a comment to explain.